src/main/java/seedu/recruittrackpro/model/RecruitTrackPro.java

```diff
@@ -108,6 +108,10 @@ public ObservableList<Person> getPersonList() {
         return persons.asUnmodifiableObservableList();
     }
 
+    public UniquePersonList getUniquePersonList() {
+        return persons;
+    }
+
```

Comment:
This breaks the encapsulation since `persons` is set to `private`. As per earlier suggestion to resolve the violation of Law of Demeter, consider adding the following methods which allows `RecruitTrackPro` to operate on its internal `persons` object without exposing it.

```suggestion
public void togglePersonListSorting() {
    persons.switchSorting();
}

public boolean isPersonListSortedAscending() {
    return persons.isAscending();
}